server/core/src/indexer/indexer.ts

```diff
@@ -402,7 +402,8 @@ export class FullTextIndexPipeline implements FullTextPipeline {
                   limit: globalIndexer.processingSize,
                   sort: {
                     _id: 1
-                  }
+                  },
+                   total: true
```

Comment:
This is not a good idea, total is very slow.

Reply:
fixed